apps/nerves_hub_www/lib/nerves_hub_www_web/templates/device/index.html.leex

```diff
@@ -24,56 +24,77 @@
     <thead>
       <tr>
         <%= devices_table_header("Identifier", "identifier", @current_sort, @sort_direction) %>
-        <th>Firmware Info</th>
-        <%= devices_table_header("Status", "status", @current_sort, @sort_direction) %>
-        <%= devices_table_header("Last Connection", "last_communication", @current_sort, @sort_direction) %>
-        <%= devices_table_header("Tags", "tags", @current_sort, @sort_direction) %>
+        <%= devices_table_header("Connection", "status", @current_sort, @sort_direction) %>
+        <%= devices_table_header("Last Handshake", "last_communication", @current_sort, @sort_direction) %>
+        <th>Firmware</th>
+        <%= devices_table_header("Groups", "tags", @current_sort, @sort_direction) %>
         <th></th>
       </tr>
     </thead>
     <%= for device <- @devices do %>
       <tr class="item">
         <td>
-          <a href="<%= Routes.device_path(@socket, :show, @org.name, @product.name, device.identifier) %>" class="">
+          <a href="<%= Routes.device_path(@socket, :show, @org.name, @product.name, device.identifier) %>" class="badge healthy-<%= device.healthy %>">
             <%= device.identifier %>
           </a>
         </td>
+
+        <td>
+          <div class="ml-3">
+          <%= if device.status == "offline" do %>
```

Comment:
Do we know what all of the possible statuses are? I wonder if there is a possibility that the device might not be online but have a different status like "error" or something.